packages/entities/entities-shared/docs/entity-link.md

```diff
@@ -0,0 +1,66 @@
+# EntityLink.vue
+
+A link component that displays label text, and optionally links to an absolute URL. Used for navigation between micro-frontends.
+
+- [Requirements](#requirements)
+- [Usage](#usage)
+  - [Install](#install)
+  - [Props](#props)
+  - [Usage example](#usage-example)
+- [TypeScript interfaces](#typescript-interfaces)
+
+## Requirements
+
+- `vue` must be initialized in the host application
+- `@kong/kongponents` must be added as a `dependency` in the host application, globally available via the Vue Plugin installation, and the package's style imports must be added in the app entry file. [See here for instructions on installing Kongponents](https://kongponents.konghq.com/#globally-install-all-kongponents).
+
+## Usage
+
+### Install
+
+[See instructions for installing the `@kong-ui-public/entities-shared` package.](../README.md#install)
+
+### Props
+
+#### `entityLinkData`
+
+- type: `Object as PropType<EntityLinkData>`
+- required: `true`
+
+Metadata associated with the link.
+
+#### `externalLink`
+
+- type: `String`
+- required: `false`
+- default: `''`
+
+The absolute URL to be used as the `href` attribute.
+
+#### `newWindow`
+
+- type: `Boolean`
+- required: `false`
+- default: `true`
```

Comment:
question: is there any way we could default the boolean props to `false` so usage will always be ex. `new-window` instead of always `:new-window="false"`

Reply:
Good point. there are literally only two places to update in consuming apps, so this is preferable :)